backend/python/migrations/versions/5a1dbdb0d816_add_email_to_user_table.py

```diff
@@ -5,26 +5,25 @@
 Create Date: 2021-10-13 02:19:05.769253
 
 """
-from alembic import op
```

Comment:
this is just some linting changes i believe! alembic migrations unfortunately do not conform with black :(

the circle ci linting only checks code in `backend/app` so that's why it didn't fail on your PR--do you want me to change it to lint the migration files as well?

Reply:
I trust your judgement in this area so I'll leave the ball in your park :)